src/scenes/end.js

```diff
@@ -1,23 +1,82 @@
-import elements from "../elements.js";
+import options from "../options.js";
 
 export default class EndScene extends Phaser.Scene {
     constructor() {
         super({ key: "EndScene" });
+        this.gameReplay = {};
+        this.buildYourOwn = {};
```

Comment:
I wouldn't recommend setting default values like this in the constructor unless those values will be used: which these are not. Here and in the other scenes.

IMO it's better to have no data then an inconsistent data shape.